packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -66,8 +68,9 @@ export interface EnqueueLinksOptions {
      *
      * An array of {@link PseudoUrl} strings or plain objects
```

Comment:
We no longer use this class, so it won't be present in the docs and the link most probably won't work. Maybe we should move the content from the [JSDoc of that class](https://github.com/apify/apify-shared-js/blob/master/packages/pseudo_url/src/pseudo_url.ts#L4) to some other place so it gets rendered in our docs - we dont need the full content as is, but we should keep the part that describes what PURL is and how it works somewhere (could be a separate docs page, doesn't have to be in code/jsdoc).

But I guess we don't have to care

Reply:
nope, it wont be part of the monorepo

Reply:
Welp :] Then let's go with your plan above =D